service/src/main/java/org/jboss/sbomer/service/feature/sbom/service/AdvisoryService.java

```diff
@@ -663,20 +662,6 @@ protected Collection<SbomGenerationRequest> createBuildManifestsForRPMBuilds(
         return sbomRequests;
     }
 
-    private Long findErrataProductVersionIdByName(ErrataProduct product, String pVersionName) {
```

Comment:
This method was unused. I just removed it.

Reply:
sounds good